docs/api/browser-window.md

```diff
@@ -983,6 +983,29 @@ Same as `webContents.capturePage([rect, ]callback)`.
   * `httpReferrer` String - A HTTP Referrer url.
   * `userAgent` String - A user agent originating the request.
   * `extraHeaders` String - Extra headers separated by "\n"
+  * `postData` Array - An array of `upload` objects which
```

Comment:
I _think_ @MarshallOfSound was suggesting a markdown file for each upload type, e.g. `structures/upload-data.md`, `structures/upload-file.md`, etc.